### DISCUSSION TITLE
remember the cursor position  in TextField

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [microsoft/fluentui#26512] [Q&A] [answered]
Is there a way to get the cursor position  onchange of TextField  and set it back when user comes back to it?

Thank you

**Comment by layershifter** [comment, marked as answer]
@Ofer-Gal there is `setSelectionRange()` API and properties on `input` elements:

> This method updates the HTMLInputElement.selectionStart, selectionEnd, and selectionDirection properties in one call.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/setSelectionRange
https://codesandbox.io/s/cool-darkness-cwe7oh?file=/src/App.js
